hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoCreateHandler.java

```diff
@@ -110,6 +129,17 @@ public void handle(@NonNull final HandleContext handleContext) {
         final var txnBody = handleContext.body();
         final var op = txnBody.cryptoCreateAccount();
 
+        final var fees = handleContext.feeCalculator(SubType.DEFAULT).legacyCalculate(sigValueObj -> {
+            try {
+                final var protoBody = fromPbj(handleContext.body());
+                return CryptoFeeBuilder.getCryptoCreateTxFeeMatrices(protoBody, sigValueObj);
+            } catch (InvalidTxBodyException e) {
+                throw new RuntimeException(e);
```

Comment:
Could we put a `@todo('post-modularization-cleanup') Replace this with an appropriate HandleException` comment on these cases where we throw RuntimeException only because we need to match current behavior?

Reply:
I usually add `FUTURE` for it, added. Thanks